doc/styles/Button.md

```diff
@@ -0,0 +1,158 @@
+# Button Control
+## Styles
+Style Key|IsDefaultStyle*
+-|-
+ElevatedButtonStyle|
+FilledButtonStyle|True
+FilledTonalButtonStyle|
+OutlinedButtonStyle|
+TextButtonStyle|
+IconButtonStyle|
+
+IsDefaultStyle*: Styles in this column will be set as the default implicit style for the matching control
```

Comment:
```suggestion
IsDefaultStyle*: Styles in this column will be set as the default implicit style for the matching control.
```

Here and all occurrences.

Reply:
Alternatively use a special character and replace the duplication of `IsDefaultStyle*` with the special char:

```suggestion
<sup>§</sup> Styles in this column will be set as the default implicit style for the matching control
```

Reply:
In any case this should be applied to all such occurrences.

Reply:
I think the asterisk is good enough. I've only seen § is law documents.
While I dont have a particular affinity for these, the switch would add a lot of unnecessary works.